core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -710,28 +710,28 @@ trunc_f64_to_int(WASMModuleInstance *module, uint32 *frame_sp, float64 src_min,
             CHECK_BULK_MEMORY_OVERFLOW(addr + offset, 1, maddr);     \
             CHECK_ATOMIC_MEMORY_ACCESS();                            \
                                                                      \
-            os_mutex_lock(&node->shared_mem_lock);                   \
+            shared_memory_lock(memory);                              \
```

Comment:
An concern is about the performance, calling `shared_memory_lock/unlock` adds two function calls, it may impact the performance. How about getting the lock pointer, and here `os_mutex_{lock|unlock}(shared_mem_lock)`?

Reply:
does it matter? if we really care the performance here, i guess we should use atomic ops as we do for aot.

Reply:
I am not very sure, the performance may be not so important here, I just see an opportunity to refine the code😊. Using atomic operations may be also a choice in the future.
@loganek, any idea?